docs/a320-simvars.md

```diff
@@ -1328,6 +1328,13 @@ In the variables below, {number} should be replaced with one item in the set: {
     - Bool
     - Whether or not the GPS is used as the primary means of navigation/position determination.
 
+## Flight Management System
+
+- A32NX_FM_ENABLE_APPROACH_PHASE
+  - Bool
+  - Indicates whether the FMS should switch to APPROACH phase.
+  - **WARNING:** This is temporary and internal. Do not use.
```

Comment:
If a simvar is temporary or private then it should not appear in this .md

Reply:
This doc is for us developers right? The public docs on the website shouldn't have internal/private vars.